evaluator/src/main/kotlin/Rule.kt

```diff
@@ -102,24 +107,31 @@ abstract class Rule(
      */
     abstract fun issueSource(): String
 
-    fun issue(severity: Severity, message: String) {
-        issues += OrtIssue(source = issueSource(), severity = severity, message = message)
+    fun issue(severity: Severity, pkgId: Identifier, license: String?, licenseSource: LicenseSource?, message: String) {
+        violations += RuleViolation(
+            severity = severity,
+            rule = name,
+            pkg = pkgId,
+            license = license,
+            licenseSource = licenseSource,
+            message = message
+        )
     }
 
     /**
      * Add a [hint][Severity.HINT] to the list of [issues].
      */
-    fun hint(message: String) = issue(Severity.HINT, message)
+    fun hint(pkgId: Identifier, license: String?, licenseSource: LicenseSource?, message: String) = issue(Severity.HINT, pkgId, license, licenseSource, message)
```

Comment:
These lines are too long and need to be wrapped.

Reply:
Fixed.